end-to-end-tests/project.clj

```diff
@@ -3,11 +3,12 @@
   :url "http://example.com/FIXME"
   :license {:name "Eclipse Public License"
             :url  "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.9.0-RC1"]
+  :dependencies [[org.clojure/clojure "1.10.1"]
                  [clj-http "3.4.1"]
                  [cheshire "5.7.0"]
-                 [org.postgresql/postgresql "42.0.0"]
+                 [org.postgresql/postgresql "42.2.8"]
                  [org.clojure/java.jdbc "0.5.8"]
                  [listora/again "0.1.0"]]
   :repl-options {:host    "0.0.0.0"
-                 :port    47480})
+                 :port    47480}
+  :profiles {:repl {:plugins [[cider/cider-nrepl "0.21.1"]]}})
```

Comment:
Doesn't this depend on the version of cider that you have installed in your Emacs?

Reply:
I can remove that..  I don't use profiles in my `$HOME/.lein`